tests/unit/InputToEntity/MinimalItemInputTest.php

```diff
@@ -99,4 +100,72 @@ public function testGetItem( array $requestEntity, Item $expected ) {
 		);
 	}
 
+	public function testStatementsHavePropertyKey() {
+		$requestEntityMissingProperty = [
+			'wikibasereconcileedit-version' => '0.0.1/minimal',
+			'labels' => [ 'en' => 'en-label' ],
```

Comment:
We could also leave out the labels, descriptions, aliases, and sitelinks, to keep the test cases smaller.

Reply:
True :)

Reply:
Can't find this test anymore.

Reply:
Test should be back now. Please re-check :)

Reply:
I don’t see any added tests in `MinimalItemInputTest` (only `ReconciliationServiceTest`).

Reply:
Tests are here: https://github.com/wmde/WikibaseReconcileEdit/pull/25/commits/2fa36d7111ba28c906d7b8ecf4d2fc3232b05855#diff-59286771e823077ff1d0248e0248382e9ebdab648bae474dcf31bf61aeb817d3